src/subdomains/user/application/services/lightning-wallet.service.ts

```diff
@@ -74,12 +74,28 @@ export class LightningWalletService {
 
   @Cron(CronExpression.EVERY_5_MINUTES)
   @Lock()
-  async processSyncLightningUserTransactions(): Promise<void> {
+  async process1SyncLightningUserTransactions(): Promise<void> {
     if (Config.processDisabled(Process.UPDATE_LIGHTNING_USER_TRANSACTION)) return;
 
     await this.syncLightningUserTransactions(undefined, undefined, undefined, true);
   }
 
+  @Cron(CronExpression.EVERY_DAY_AT_6AM)
+  @Lock()
+  async process2SyncLightningUserTransactions(): Promise<void> {
+    if (Config.processDisabled(Process.SYNC_LIGHTNING_USER_TRANSACTIONS)) return;

```

Comment:
Maybe we could use a more meaningful naming? Like `syncRecentTransactions` and `syncAllTransactions`?